src/main/java/prgms/vouchermanagementapp/controller/WebVoucherController.java

```diff
@@ -0,0 +1,89 @@
+package prgms.vouchermanagementapp.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+import prgms.vouchermanagementapp.domain.Voucher;
+import prgms.vouchermanagementapp.domain.dto.VoucherViewDTO;
+import prgms.vouchermanagementapp.service.VoucherService;
+
+import java.util.List;
+import java.util.UUID;
+
+@Controller
+@RequestMapping("/voucher/voucherList")
+public class WebVoucherController {
```

Comment:
```suggestion
@Controller
@RequestMapping("/vouchers")
public class WebVoucherController {
```

지금은 SSR라서 약간 제한될 수 있지만, 최대한 REST API를 지켜보는게 좋겠네요
https://meetup.toast.com/posts/92